orca-api/src/main/java/com/netflix/spinnaker/orca/api/pipeline/TaskExecutionInterceptor.java

```diff
@@ -14,10 +14,8 @@
  * limitations under the License.
  */
 
-package com.netflix.spinnaker.orca;
+package com.netflix.spinnaker.orca.api.pipeline;
 
-import com.netflix.spinnaker.orca.api.pipeline.Task;
-import com.netflix.spinnaker.orca.api.pipeline.TaskResult;
 import com.netflix.spinnaker.orca.api.pipeline.models.StageExecution;
 import java.util.concurrent.TimeUnit;
```

Comment:
Sorry - more stuff:

- Could you mark this interface as `@Beta`?
- Any reason not to have it extend `SpinnakerExtensionPoint`?
- Could you add doc comments to the methods that aren't documented?

Reply:
I don't think there is any reason to not have it extend `SpinnakerExtensionPoint`, thats a good idea. I'll also mark it as `@Beta` and add doc comments. 👍